### ISSUE TITLE
Add RSS feed to the Blog page

### ISSUE THREAD

**Issue by darkred** [violentmonkey/violentmonkey.github.io#22]
I believe it would be useful to have RSS feed for the Blog page https://violentmonkey.github.io/posts/ .
The posts there are few but all very useful and important, must-read.

If it helps, [this guide](https://dzhavat.github.io/2020/01/19/adding-an-rss-feed-to-github-pages.html) says you can just: 
- add the GitHub pages plugin [jekyll-feed](https://github.com/jekyll/jekyll-feed) to the plugins array in the `_config.yml` file 
- add this line in the head of the HTML file:
`<link rel="alternate" type="application/atom+xml" title="Violentmonkey’s blog" href="/feed.xml">`
- finally, make a link to the feed that is accessible from the page, like this:
`<a class="btn btn-rss" href="/feed.xml" target="_blank">RSS</a>`

**Comment by gera2ld** [comment]
I thought RSS was used to track updates so it seems useless to provide RSS for posts that are rarely updated.

It only makes sense to me if you are using RSS to keep a list of posts as what bookmarks do. If that's the case, I think we have better choices. Power users who are interested in implementation details can easily find the site of Violentmonkey in the popup and read the posts, and other users may not be interested in them.

Closing since it's not really useful. Feel free to reopen if you have more ideas.

**Comment by darkred** [comment]
Yes, the posts are rarely updated, they are 4 in total, three made in '17 and the other in '18 .

From the 4 blog posts in total, 3 there are links to 3 for them in VM's settings.
So, yes, a power user can discover almost all posts just by browsing the settings (and expanding the Advanced section), and the New script(editor) page.

On the other hand, the popup links to the the main page https://violentmonkey.github.io/ ,
but the blog post titles are not displayed in there,
and so, if someone misses the Blog link (located in the far top right corner, easy to miss, I think), he will probably miss the posts.

I use RSS reader (QuiteRSS) in order to easily get updates of multiple sites, lots of them.
I keep it running constantly, and so I never miss updates.

---

To summarize, ideally for me would be to add the titles of the posts to the main page + add RSS feed.
I would be glad though if you could just add the titles to the main page.
